packages/devtools_app/lib/src/screens/debugger/program_explorer_controller.dart

```diff
@@ -182,6 +182,17 @@ class ProgramExplorerController extends DisposableController
     }
   }
 
+  void resetOutline() {
```

Comment:
Can you add a comment describing what this method does?

Reply:
Sure!